### DISCUSSION TITLE
Change UI Admin website

### DISCUSSION THREAD

**Discussion by hiensarahly** [walinejs/waline#107] [Q&A] [answered]
I want to change the ui admin personal website at /ui/register, how can I change it?

**Comment by lizheming** [comment, marked as answer]
It has no custom configure can change admin dashboard right now. Admin dashboard is a Single Page Application, and source code is here https://github.com/lizheming/waline/tree/master/packages/admin. Admin dashboard use cdn url to load it, you can change here https://github.com/lizheming/waline/blob/master/packages/server/src/middleware/dashboard.js#L16. It need you have some web program skill if you want to change it.

**Reply by hiensarahly** [reply]
I mean not the admin dashboard, but this website here:

[![42-A4-BD78-8928-420-B-A128-FCDDF574109-E.jpg](https://i.postimg.cc/fy7Tkz5x/42-A4-BD78-8928-420-B-A128-FCDDF574109-E.jpg)](https://postimg.cc/0KQqBqTQ)

**Reply by lizheming** [reply]
Oh, the personal website data is not used right now, and have no place to change it. If you really want to change it you can go to your storage change data directly.

**Comment by hiensarahly** [comment]
Ok, that’ll work! Thank u 😇

**Reply by lizheming** [reply]
you 2 welcome~

**Comment by lizheming** [comment]
Hi @hiensarahly, I have added `/ui/profile` page that can modify personal website data now. You can redirect to here by clicking username at top right corner. Thank you for your advice~

![image](https://user-images.githubusercontent.com/424491/103471394-c5a34700-4dba-11eb-88ef-5f8699ac1e2b.png)
